docs/pages/config/database/evm-json-rpc-cache.mdx

```diff
@@ -125,35 +127,36 @@ database:
     
     # Define caching policies for different network/method/finality states
     policies:
-      # Example: Cache all methods with finalized data including empty responses
+      # Example: Cache realtime data only for 2 seconds (eth_blockNumber, eth_gasPrice, etc) to reduce costs yet fresh enough data
       - network: "*"
         method: "*"
-        finality: finalized
-        empty: allow
+        finality: realtime
+        empty: ignore
         connector: memory-cache
-        ttl: 0
+        ttl: 2s
       # Example: Cache unfinalized data only for 5 seconds (getLogs of a recent block) except empty responses
       - network: "*"
         method: "*"
         finality: unfinalized
         empty: ignore
         connector: memory-cache
-        ttl: 5s
-      # Example: Cache unknown finalization data (eth_trace*) only for 10 seconds
+        ttl: 10s
```

Comment:
these changes seem like they belong in another commit

Reply:
this is actually my OCD reorging items in this docs page to apply latest recommended cache settings :D